packages/fileimport-service/ifc/import_file.js

```diff
@@ -1,4 +1,5 @@
 const fs = require('fs')
+const { fileimportServiceLogger } = require('../observability/logging')
```

Comment:
if there's no other kind of logger (in this package), I'd vote for just calling it `logger`